contrib/pod-master/Makefile

```diff
@@ -0,0 +1,16 @@
+all: push
+
+# Set this to the *next* version to prevent accidentally overwriting the existing image.
+TAG = 1.2
```

Comment:
Suggest taking TAG on the command line-- no reason to keep it checked in like this.

e.g. ```$ make push TAG=1.3```